build-shims.js

```diff
@@ -78,7 +78,8 @@ mockery.registerMock('@zimbra-client/enhancers', {
 	withMediaQuery: 1,
 	withTabManager: 1,
 	withIntlWrapper: 1,
-	clientConfiguration: 1
+	clientConfiguration: 1,
+	withTracking: 1
```

Comment:
I think we want to provide this decorator option for zimlets which are developed with class-based components. But the developer can wrap a component and use hook same time to achieve same functionality.